graphdatascience/tests/integration/test_graph_object.py

```diff
@@ -37,6 +37,14 @@ def G(gds: GraphDataScience) -> Generator[Graph, None, None]:
     gds.graph.drop(G, False)
 
 
+def test_graph_database(gds: GraphDataScience, G: Graph) -> None:
+    assert G.database() == gds.database()
+
+
+def test_graph_configuration(G: Graph) -> None:
+    assert "Node" in G.configuration()["nodeProjection"].keys()
```

Comment:
very superficial test, but I think this is enough. we have plenty of tests on the server side